src/include/daos_srv/daos_server.h

```diff
@@ -535,81 +535,12 @@ int ds_obj_fetch(daos_handle_t oh, daos_epoch_t epoch,
 		 daos_key_t *dkey, unsigned int nr,
 		 daos_iod_t *iods, d_sg_list_t *sgls,
 		 daos_iom_t *maps);
-int ds_obj_list_obj(daos_handle_t oh, daos_epoch_t epoch, daos_key_t *dkey,
+int ds_obj_list_obj(daos_handle_t oh, daos_epoch_t *epoch, daos_key_t *dkey,
```

Comment:
sorry, I still cannot find where did you set the returned epoch value?
you set the oeo_epoch at server handler, but seems no place to get the value to assign back to *epoch.

Reply:
Sorry, I missed to reassign the epoch back to the RPC sponsor. I will refresh the patch. Thanks for catching that.